### PR TITLE
Various functions

### PR DESCRIPTION
**Pull request by olakiril**
MaskCoordinates & ScanCoordinates extract information from the reader and should be in the preprocess structure. 

tuning RFs implement most things in matlab side

### CODE REVIEW

File: matlab/schemas/+anatomy/MaskCoordinates.m [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                'px_width','px_height','um_width','um_height');
+            depth = fetch1(experiment.Scan & key, 'depth');
+            
+            for imask = 1:length(mask_id);
+                
+                % get mask position
+                mask = masks{imask};
+                im = zeros(px_width,px_height);
+                im(mask) = weights{imask};
+                labeledImage = logical(true(size(im)));
```
Comment (dimitri-yatsenko) [inline matlab/schemas/+anatomy/MaskCoordinates.m]: `logical` is unnecessary here.

File: matlab/schemas/+anatomy/MaskCoordinates.m [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                measurements = regionprops(labeledImage, im, 'WeightedCentroid');
```
Comment (dimitri-yatsenko) [inline matlab/schemas/+anatomy/MaskCoordinates.m]: this does not seem like it should work since `labeledImage` is all true.

Comment (olakiril) [inline matlab/schemas/+anatomy/MaskCoordinates.m]: It does since the mask is in im

Comment (dimitri-yatsenko) [inline matlab/schemas/+anatomy/MaskCoordinates.m]: but that's not how it works, according to the documentation.  Have you tested this?  The mask is in the first image and the weights are in the second.

Comment (olakiril) [inline matlab/schemas/+anatomy/MaskCoordinates.m]: It is how it works. the first argument provides the information about the different labeled regions, in which case it is all just one region, and the second argument is the image. The two have to have the same size.

**Pull request merged**